### PR TITLE
add pbb to Dropout/SpatialDropout tostring method

### PR DESCRIPTION
**Pull request by orian**
This makes Dropout more similar to Linear:

```
  (1): nn.SpatialConvolution
  (2): nn.ReLU
  (3): nn.SpatialConvolution
  (4): nn.ReLU
  (5): nn.SpatialDropout(0.4)
  (6): nn.View
  (7): nn.Linear(376360 -> 200)
```

### CODE REVIEW

**Comment by soumith** [comment]
Could you please squash your commits to a single commit. Thanks for the PR

**Comment by orian** [comment]
Sure, I will send it later today, maybe add something similar for SpatialConvolution and MaxPooling
